src/components/templates/modal/modal.module.scss

```diff
@@ -18,5 +18,5 @@
   margin: 1rem;
   border-radius: 1rem;
   cursor: initial;
-  overflow-y: scroll;
+  overflow-y: auto;
```

Comment:
e don't have unit tests. Have you manually tested places that are using modals? Changing this might introduce new bugs. Please ensure other places work as expected.